### PR TITLE
Add utils project

### PR DESCRIPTION
**Pull request by grorge123**
- Create a Rust program to generate ETH key pairs
- Create a Rust program to update the registration smart contract

### CODE REVIEW

**Review by juntao — COMMENTED** [review]
Thanks. Can you create a folder `utils/registry` for this project? That is because we will have multiple utils like this.

**Comment by grorge123** [comment]
> Thanks. Can you create a folder `utils/registry` for this project? That is because we will have multiple utils like this.

I have modified it. Please check, thx.

**Comment by juntao** [comment]
Is it possible to generate the private key into a keystore file? The `config.json` file could store the keystore password instead?

In this way, the `config.json` file starts with a `password` field and an empty `address` field.

**Comment by grorge123** [comment]
> Is it possible to generate the private key into a keystore file? The `config.json` file could store the keystore password instead?
> 
> In this way, the `config.json` file starts with a `password` field and an empty `address` field.

I have added it. Please check, thx.

File: utils/registry/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[package]
+name = "utils"
```
Comment (juntao) [inline utils/registry/Cargo.toml:2]: Let's call it `registry.wasm`

Comment (grorge123) [reply utils/registry/Cargo.toml]: I have changed it. Because '.' is invalid character, I replaced it with '_'.

**Review by juntao — APPROVED** [review]
I made a direct change to fix the crate name. Thanks!

**Comment by juntao** [comment]
@grorge123 Let's go one step further -- if the password field is missing, we will generate a random password and put into config.json.

Please open another PR for that. Thanks.

**Pull request merged**